paper/paper.md

```diff
@@ -32,22 +32,20 @@ We present E2EDNA 2.0: End-2-End DNA 2.0, a python simulation pipeline which off
-
-# Figure
+The gap that the E2EDNA family of programs addresses is the absence of a non-black-box one-stop-shop aptamer simulation package, which is capable of providing in silico predictions of 2D structure, 3D structure, and aptamer-ligand binding simulation while keeping methodology flexible and analysis open-ended. E2EDNA 2.0 achieves this in two key ways: a python-interfacing implementation which makes installation and access easier for users than the original E2EDNA package, and the automation of the dozens of tasks required in setting up, running, and interpreting atomistic aptamer-ligand simulations.
 
 ![Schematic workflow of E2EDNA 2.0 pipeline.\label{fig:figure_workflow}](figure_workflow.png){ width=100% }
 
 
 # Components and Features
 
-As shown in \autoref{fig:figure_workflow}, the complete simulation pipeline in E2EDNA 2.0 consists of the following main steps: 1. Secondary structure prediction, 2. Tertiary structure prediction, 3. Molecular dynamics simulation, and 4. Aptamer-ligand docking. The key inputs into E2EDNA 2.0 are the DNA sequence in the FASTA format, the structure of the ligand in the pdb format (optional), and the choice of simulation mode. The output includes secondary structure in dot-bracket notation, tertiary structures in pdb format of free aptamer and aptamer-ligand complex (optional), and simulation trajectories in the dcd format. It is worth noting that there are other parameters besides "key inputs" that can be customized in the main.py, such as solvent ionic strength. Detailed analysis of the generated trajectories is not performed by E2EDNA 2.0, though is straightforward to set-up for a particular workflow using built-in or user-specified functions.
+As shown in \autoref{fig:figure_workflow}, the complete simulation pipeline in E2EDNA 2.0 consists of the following main steps: 1. Secondary structure prediction, 2. Tertiary structure prediction, 3. Molecular dynamics simulation, and 4. Aptamer-ligand docking. The key inputs into E2EDNA 2.0 are the DNA sequence in the FASTA format, the structure of the ligand in the pdb format (optional), and the choice of simulation mode. The output includes secondary structure in dot-bracket notation, tertiary structures in pdb format of free aptamer and aptamer-ligand complex (optional), and simulation trajectories in the dcd format. It is worth noting that there are other parameters besides "key inputs" that can be customized in main.py, such as solvent ionic strength. Detailed analysis of the generated trajectories is not performed by E2EDNA 2.0, though is straightforward to set up for a particular workflow using built-in or user-specified functions.
```

Comment:
After merging the commits, I will further delete the phrase "in the main.py" because we have modified the code to allow users to change pipeline settings without touching the source code, thanks to the two reviewers' suggestions.